src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -247,6 +272,44 @@ public synchronized ProcessWrapper launchTranscode(
 		return pw;
 	}
 
+	/**
```

Comment:
Damned Github, i cannot see anything except` /**`

Please, feel free to push a commit that i can understand what you mean, for an eventual next time.